memgpt/main.py

```diff
@@ -304,7 +304,8 @@ async def main(
     chosen_persona = cfg.memgpt_persona
 
     memgpt_agent = presets.use_preset(
-        presets.DEFAULT,
+        # presets.DEFAULT,
+        "memgpt_extras",

```

Comment:
Need to rebase on main and test that we can launch the "extras" preset using the preset flag/config.